backend/open_webui/test/apps/webui/routers/test_artifacts_redaction.py

```
@@ -884,3 +884,96 @@ def _gif_transparency_ocr(image_path, ocr_reader=None, ocr_engine="easyocr"):
+    module_name = "backend.open_webui.retrieval.loaders.ocrprocessor"
+    project_root = Path(__file__).resolve().parents[6]
+    monkeypatch.syspath_prepend(str(project_root))
+    existing = sys.modules.get(module_name)
+    if existing is not None:
+        return existing
+
+    if "easyocr" not in sys.modules:
+        easyocr_stub = types.ModuleType("easyocr")
+
+        class _StubReader:
+            def __init__(self, *args, **kwargs):
```

Ruff (0.13.1)

Unused method argument: `args`

(ARG002)